CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* Add `ProjectRemoteState` attribute to `Workspace` by @hs26gill [#1248](https://github.com/hashicorp/go-tfe/pull/1248)
```

Comment:
I'm struggling to author a CHANGELOG entry that is both accurate and useful given the timing around merging the ultimate terraform-provider-tfe changes that depend on this PR. What do you think about this?

```suggestion
* The Workspace `ProjectRemoteState` attribute is now available in Terraform Enterprise, and will soon be available in HCP Terraform, by @hs26gill [#1248](https://github.com/hashicorp/go-tfe/pull/1248)
```